_docs/collators.md

```diff
@@ -139,3 +184,23 @@ For example, in Turing, if you've joined as a collator candidate, and you are co
 }
 ```
 Because you opted to join sometime during this round, you can calculate the `FIRST_BLOCK_NUMBER_OF_ROUND + 1800`, which would mean you can expect to start producing blocks at `202,537`.
+
+### When will my new session keys be used?
+New session keys are used after the current session completes and two full sessions elapse.
+
+### Under what circumstances would I be removed from the candidate pool?
+Currently slashing has not been implemented on the Turing network. You will only be removed from the candidate pool if you initiate leaving the pool through extrinsic.
+
+### I want to run multiple collators, can I use the same wallet?
+No. Each collator must be associated with a unique wallet in order to join the candidate pool. 
+
+### What rewards will I get?
+Collators collect 2 types of rewards: the collator reward and the staking reward. 1% annual inflation goes directly to collators. This does not dilute based on the number of people staking to your node. 2.5% annual inflation goes to staking rewards. This is split proportionately among all stakers to that node. This will be diluted as more people stake to your node. This creates an incentive for roughly all the nodes to have similar stakes for stakers to maximize their APY.
```

Comment:
These numbers are based off of the current inflation (5%). If we ever change that these will become inaccurate.

If we want to decouple from the 5% inflation, we could present these numbers differently- 20% of the annual inflation goes to collators, 50% of the annual inflation goes to staking rewards.

My suggestion is easier to maintain in the long term, but if you think this is easier for users to understand in the moment then I'm good with the current text.

Reply:
I'll do a separate PR for this